<!DOCTYPE html>
<html class="html" lang="en-US">
 <head>

  <script type="text/javascript">
   if(typeof Muse == "undefined") window.Muse = {}; window.Muse.assets = {"required":["jquery-1.8.3.min.js", "museutils.js", "jquery.musemenu.js", "jquery.watch.js", "research.css"], "outOfDate":[]};
</script>
  
  <meta http-equiv="Content-type" content="text/html;charset=UTF-8"/>
  <meta name="generator" content="2015.0.2.310"/>
  <link rel="shortcut icon" href="images/favicon.ico?3879086630"/>
  <title>Research</title>
  <!-- CSS -->
  <link rel="stylesheet" type="text/css" href="css/site_global.css?68857643"/>
  <link rel="stylesheet" type="text/css" href="css/master_a-master.css?199581093"/>
  <link rel="stylesheet" type="text/css" href="css/research.css?187960283" id="pagesheet"/>
  <!-- Other scripts -->
  <script type="text/javascript">
   document.documentElement.className += ' js';
var __adobewebfontsappname__ = "muse";
</script>
  <!-- JS includes -->
  <script type="text/javascript">
   document.write('\x3Cscript src="' + (document.location.protocol == 'https:' ? 'https:' : 'http:') + '//webfonts.creativecloud.com/open-sans:n4:default.js" type="text/javascript">\x3C/script>');
</script>
  <!--[if lt IE 9]>
  <script src="scripts/html5shiv.js?4241844378" type="text/javascript"></script>
  <![endif]-->
   </head>
 <body>

  <div class="shadow clearfix" id="page"><!-- column -->
   <div class="position_content" id="page_position_content">
    <div class="clearfix pinned-colelem" id="u322-3"><!-- content -->
     <p>&nbsp;</p>
    </div>
    <nav class="MenuBar clearfix pinned-colelem" id="menuu136"><!-- horizontal box -->
     <div class="MenuItemContainer clearfix grpelem" id="u137"><!-- vertical box -->
      <a class="nonblock nontext MenuItem MenuItemWithSubMenu clearfix colelem" id="u138" href="index.html"><!-- horizontal box --><div class="MenuItemLabel NoWrap clearfix grpelem" id="u139-4"><!-- content --><p>GCA Lab</p></div></a>
     </div>
     <div class="MenuItemContainer clearfix grpelem" id="u151"><!-- vertical box -->
      <a class="nonblock nontext MenuItem MenuItemWithSubMenu MuseMenuActive clearfix colelem" id="u154" href="research.html"><!-- horizontal box --><div class="MenuItemLabel NoWrap clearfix grpelem" id="u156-4"><!-- content --><p>Research</p></div></a>
     </div>
     <div class="MenuItemContainer clearfix grpelem" id="u237"><!-- vertical box -->
      <a class="nonblock nontext MenuItem MenuItemWithSubMenu clearfix colelem" id="u238" href="resources.html"><!-- horizontal box --><div class="MenuItemLabel NoWrap clearfix grpelem" id="u240-4"><!-- content --><p>Resources</p></div></a>
     </div>
     <div class="MenuItemContainer clearfix grpelem" id="u165"><!-- vertical box -->
      <a class="nonblock nontext MenuItem MenuItemWithSubMenu clearfix colelem" id="u168" href="publications.html"><!-- horizontal box --><div class="MenuItemLabel NoWrap clearfix grpelem" id="u171-4"><!-- content --><p>Publications</p></div></a>
     </div>
     <div class="MenuItemContainer clearfix grpelem" id="u144"><!-- vertical box -->
      <a class="nonblock nontext MenuItem MenuItemWithSubMenu clearfix colelem" id="u145" href="courses.html"><!-- horizontal box --><div class="MenuItemLabel NoWrap clearfix grpelem" id="u146-4"><!-- content --><p>Courses</p></div></a>
     </div>
     <div class="MenuItemContainer clearfix grpelem" id="u158"><!-- vertical box -->
      <a class="nonblock nontext MenuItem MenuItemWithSubMenu clearfix colelem" id="u161" href="lab-members.html"><!-- horizontal box --><div class="MenuItemLabel NoWrap clearfix grpelem" id="u164-4"><!-- content --><p>Lab Members</p></div></a>
     </div>
     <div class="MenuItemContainer clearfix grpelem" id="u172"><!-- vertical box -->
      <a class="nonblock nontext MenuItem MenuItemWithSubMenu clearfix colelem" id="u173" href="links.html"><!-- horizontal box --><div class="MenuItemLabel NoWrap clearfix grpelem" id="u176-4"><!-- content --><p>Links</p></div></a>
     </div>
    </nav>
    <a class="nonblock nontext pinned-colelem" id="u226" href="index.html"><!-- svg --><img class="svg" src="images/pasted%20svg%2044x50.svg" width="34" height="39" alt="" data-mu-svgfallback="images/pasted%20svg%2044x50_poster_.png"/></a>
    <div class="clearfix colelem" id="u383-4"><!-- content -->
     <p>RESEARCH</p>
    </div>
    <div class="clearfix colelem" id="u335-360"><!-- content -->
     <p id="u335-2">Below are examples of ongoing research projects:</p>
     <p id="u335-3">&nbsp;</p>
     <p id="u335-7">1. <a class="nonblock anim_swing" href="research.html#originandevolutionribosome">Origin and evolution of the ribosome</a></p>
     <p id="u335-11">2. <a class="nonblock anim_swing" href="research.html#virusespredate">Viruses predate diversified cellular life</a></p>
     <p id="u335-15">3. <a class="nonblock anim_swing" href="research.html#luca">LUCA: The end of a planetary mega-organism</a></p>
     <p id="u335-19">4. <a class="nonblock anim_swing" href="research.html#originandevolutionmetabolic">Origin and evolution of metabolic networks</a></p>
     <p id="u335-23">5. <a class="nonblock anim_swing" href="research.html#molecularclock">Molecular clock of fold structures uncover the rise of planet oxygenation and aerobic metabolism</a></p>
     <p id="u335-27">6. <a class="nonblock anim_swing" href="research.html#untanglingbiodiversity">Untangling biodiversity</a></p>
     <p id="u335-31">7. <a class="nonblock anim_swing" href="research.html#crosstalkplantsmicroorganisms">Crosstalk during the interaction of plants and microorganisms</a></p>
     <p id="u335-32">&nbsp;</p>
     <p id="u335-33">&nbsp;</p>
     <p id="u335-35">___________________________________________________________________________________</p>
     <p id="u335-39"><span id="u335-36">​</span><span><span class="actAsInlineDiv normal_text" id="u1219"><!-- content --><a class="excludeFromNormalFlow" id="originandevolutionribosome"></a></span></span><span></span></p>
     <p id="u335-43"><span id="u335-40">​</span><span><span class="actAsInlineDiv normal_text" id="u460"><!-- content --><span class="actAsDiv clip_frame excludeFromNormalFlow" id="u454"><!-- image --><img id="u454_img" src="images/originlife.jpg" alt="" width="96" height="96"/></span></span></span><span></span></p>
     <p id="u335-45">ORIGIN AND EVOLUTION OF THE RIBOSOME</p>
     <p id="u335-46">&nbsp;</p>
     <p id="u335-48">The ribosome is the molecular complex that is responsible for protein biosynthesis. We recently uncovered its origin and history. A careful phylogenomic analysis of RNA and protein molecules that make up this massive molecular ensemble shows that the most ancient RNA structure interacted with the most ancient protein and that this triggered a coordinated accretion process that ultimately resulted in a functional ribosomal core, half-way in evolution of life and prior to cellular diversification. The origin of translation (protein synthesis) has always been described as the chicken-and-egg problem of biochemistry. Our results take an important step back in time to clarify its early replicative origin and the early interaction of proteins and nucleic acids. The coevolutionary patterns we find exist between ribosomal proteins and RNA falsify the ancient ‘RNA world’ hypothesis and places the rise of genetics late in evolution.</p>
     <p id="u335-49">&nbsp;</p>
     <p id="u335-60">• Harish and Caetano-Anollés (2012) PLoS ONE 7:e32776 (<a class="nonblock" href="https://www.ncbi.nlm.nih.gov/pubmed/22427882">abstract</a>)(<a class="nonblock" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0032776">PDF</a>)(<a class="nonblock" href="http://f1000.com/prime/14267096">F1000</a>)</p>
     <p id="u335-65">• Study of ribosome evolution challenges RNA world hypothesis (<a class="nonblock" href="http://news.illinois.edu/news/12/0312ribosome_GustavoCaetano-Anolles.html">press release</a>)</p>
     <p id="u335-70">• Caetano-Anollés (2013) Evidence against the RNA world hypothesi. NASA Astrobiology (<a class="nonblock" href="http://astrobiology2.arc.nasa.gov/focus-groups/current/origins-of-life/articles/a-coevolution-of-ribosomal-proteins-and-rna-evidence-against-the-rna-world-hypothesis/">online</a>)</p>
     <p id="u335-75">• Scharf (2012) Your grandmother was a molecule. Scientific American (<a class="nonblock" href="http://blogs.scientificamerican.com/life-unbounded/2012/03/14/your-grandmother-was-a-molecule/">online</a>)</p>
     <p id="u335-80">• Lewis (2012) New study contradicts RNA world theory. Biotechniques (<a class="nonblock" href="http://www.biotechniques.com/news/New-Study-Contradicts-RNA-World-Theory/biotechniques-327981.html?service=print">online</a>)</p>
     <p id="u335-84"><span id="u335-81">​</span><span id="u335-82"><span class="actAsInlineDiv normal_text" id="u1223"><!-- content --><a class="excludeFromNormalFlow" id="virusespredate"></a></span></span><span id="u335-83"></span></p>
     <p id="u335-86">___________________________________________________________________________________</p>
     <p id="u335-87">&nbsp;</p>
     <p id="u335-91"><span id="u335-88">​</span><span><span class="actAsInlineDiv normal_text" id="u467"><!-- content --><span class="actAsDiv clip_frame excludeFromNormalFlow" id="u461"><!-- image --><img id="u461_img" src="images/shapeimage_1.jpg" alt="" width="97" height="97"/></span></span></span><span></span></p>
     <p id="u335-93">VIRUSES PREDATE DIVERSIFIED CELLULAR LIFE</p>
     <p id="u335-94">&nbsp;</p>
     <p id="u335-96">Using structural phylogenomic methods we reconstructed the history of proteomes of viruses and cellular organisms. This universal ‘tree of life’ placed the viruses we sampled at its roots, suggesting viruses are distinct and ancient group.</p>
     <p id="u335-97">&nbsp;</p>
     <p id="u335-105">• Nasir, Kim and Caetano-Anollés (2012) BMC Evol Biol 15:156 (<a class="nonblock" href="http://www.ncbi.nlm.nih.gov/pubmed/22920653">abstract</a>)(<a class="nonblock" href="http://www.biomedcentral.com/1471-2148/12/156/abstract">PDF</a>)</p>
     <p id="u335-113">• Study of giant viruses shakes tree of life (<a class="nonblock" href="http://news.illinois.edu/news/12/0913viruses_GustavoCaetano-Anolles.html">press release</a>)(<a class="nonblock" href="http://www.nsf.gov/news/news_summ.jsp?cntn_id=125470&amp;org=NSF&amp;from=news">NSF</a>)</p>
     <p id="u335-118">• Riesenviren sin doch lebewesen. Spiegel, Wissenschaft (<a class="nonblock" href="http://www.spiegel.de/wissenschaft/natur/grosse-viren-sind-doch-lebewesen-a-856204.html#ref=rss">online</a>)</p>
     <p id="u335-119">&nbsp;</p>
     <p id="u335-123"><span id="u335-120">​</span><span id="u335-121"><span class="actAsInlineDiv normal_text" id="u1225"><!-- content --><a class="excludeFromNormalFlow" id="luca"></a></span></span><span id="u335-122"></span></p>
     <p id="u335-125">___________________________________________________________________________________</p>
     <p id="u335-126">&nbsp;</p>
     <p id="u335-130"><span id="u335-127">​</span><span><span class="actAsInlineDiv normal_text" id="u474"><!-- content --><span class="actAsDiv clip_frame excludeFromNormalFlow" id="u468"><!-- image --><img id="u468_img" src="images/bacteria%20dapi.jpg" alt="" width="96" height="96"/></span></span></span><span></span></p>
     <p id="u335-132">LUCA: THE END OF A PLANETARY MEGA-ORGANISM</p>
     <p id="u335-133">&nbsp;</p>
     <p id="u335-135">The last universal common ancestor of life (LUCA) was both simple and complex. An account of the structures and functions of this 'mega-organism' that covered the lands and the seas of primitive Earth ~3 billion years (Gy) ago revealed a functionally advanced cellular entity. It had functional ribosomal machinery capable of protein synthesis, but its RNA genome was relatively simple. The reign of this mega-organism ended with the rise of planet oxygenation (see below), which induced aerobic metabolism, lineages or organisms, and diversified life.</p>
     <p id="u335-136">&nbsp;</p>
     <p id="u335-144">• Kim and Caetano-Anolles (2011) BMC Evol Biol 11:140 (<a class="nonblock" href="http://www.ncbi.nlm.nih.gov/pubmed/21612591">abstract</a>)(<a class="nonblock" href="http://www.biomedcentral.com/1471-2148/11/140">PDF</a>)</p>
     <p id="u335-152">• Seufferheld et al. (2011) Biology Direct 6:50 (<a class="nonblock" href="http://www.ncbi.nlm.nih.gov/pubmed/21974828">abstract</a>)(<a class="nonblock" href="http://www.biology-direct.com/content/6/1/50">PDF</a>)</p>
     <p id="u335-157">• Life began with a planetary mega-organism. NewScientist (<a class="nonblock" href="http://www.newscientist.com/article/mg21228404.300-life-began-with-a-planetary-megaorganism.html">online</a>)</p>
     <p id="u335-162">• The last universal common ancestor had a complex cellular structure (<a class="nonblock" href="http://news.illinois.edu/news/11/1005LUCA_ManfredoSeufferheld_JamesWhitfield_Caetano_Anolles.html">press release</a>)</p>
     <p id="u335-163">&nbsp;</p>
     <p id="u335-167"><span id="u335-164">​</span><span id="u335-165"><span class="actAsInlineDiv normal_text" id="u1227"><!-- content --><a class="excludeFromNormalFlow" id="originandevolutionmetabolic"></a></span></span><span id="u335-166"></span></p>
     <p id="u335-169">___________________________________________________________________________________</p>
     <p id="u335-170">&nbsp;</p>
     <p id="u335-174"><span id="u335-171">​</span><span><span class="actAsInlineDiv normal_text" id="u480"><!-- content --><span class="actAsDiv clip_frame excludeFromNormalFlow" id="u475"><!-- image --><img id="u475_img" src="images/pathways.gif" alt="" width="97" height="97"/></span></span></span><span></span></p>
     <p id="u335-176">ORIGIN AND EVOLUTION OF METABOLIC NETWORKS</p>
     <p id="u335-177">&nbsp;</p>
     <p id="u335-179">The MANET database explores the evolution of biological networks For example, metabolic activities retrieved from the KEGG database (Japan) as entries in Enzyme Commission (EC) classification were linked to protein domain structures, and their evolutionary ages mapped to each enzymatic node of the metabolic network. A careful exploration of evolutionary patterns in these networks showed that certain metabolic pathways contained enzymes that were modular and had architectures that were old and widely shared among domains of life. We also revealed patterns of origin of modern metabolism. Metabolism originated in a “big bang”, with most enzymatic reactions at all levels of EC classification being associated with the nine most ancestral and widespread protein folds. We also revealed that the first enzymatic take-over of a prebiotic chemistry involved enzymatic activities important for nucleic acid components, such as the interconversion and biosynthesis of nucleotide derivatives. These results are highly significant.</p>
     <p id="u335-180">&nbsp;</p>
     <p id="u335-182">The evolution of enzymes and metabolic pathways is under intense debate and is important for systems biology. There are indications that during evolution single enzymes have been recruited from different pathways. However, pathway duplication, enzyme specialization, retro-evolution, and de novo invention may play important roles. Our ultimate goal is to understand how these processes shape the evolution of metabolic networks.</p>
     <p id="u335-183">&nbsp;</p>
     <p id="u335-191">• Caetano-Anollés and Caetano-Anollés (2013) PLoS ONE 8: e59300 (<a class="nonblock" href="http://www.ncbi.nlm.nih.gov/pubmed/23516625">abstract</a>)(<a class="nonblock" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0059300">PDF</a>)</p>
     <p id="u335-199">• Caetano-Anollés et al. (2009) Intl J Biochem Cell Biol 41:285-297 (<a class="nonblock" href="http://www.ncbi.nlm.nih.gov/pubmed/18790074">abstract</a>)(<a class="nonblock" href="http://www.sciencedirect.com/science/article/pii/S1357272508003373">PDF</a>)</p>
     <p id="u335-207">• Caetano-Anollés, Kim and Mittenthal (2007) Proc Natl Acad Sci USA 104:9358-9363 (<a class="nonblock" href="http://www.ncbi.nlm.nih.gov/pubmed/17517598">abstract</a>)(<a class="nonblock" href="http://www.pnas.org/content/104/22/9358.long">PDF</a>)</p>
     <p id="u335-215">• Kim, Mittenthal and Caetano-Anollés (2006) BMC Bioinformatics 7:351 (<a class="nonblock" href="http://www.ncbi.nlm.nih.gov/pubmed/16854231">abstract</a>)(<a class="nonblock" href="http://www.biomedcentral.com/1471-2105/7/351">PDF</a>)</p>
     <p id="u335-220">• Evolution of metabolic enzymes reveal the importance of RNA (<a class="nonblock" href="http://www.pnas.org/cgi/content/full/104/22/9101">PNAS commentary</a>)</p>
     <p id="u335-221">&nbsp;</p>
     <p id="u335-225"><span id="u335-222">​</span><span id="u335-223"><span class="actAsInlineDiv normal_text" id="u1229"><!-- content --><a class="excludeFromNormalFlow" id="molecularclock"></a></span></span><span id="u335-224"></span></p>
     <p id="u335-227">___________________________________________________________________________________</p>
     <p id="u335-228">&nbsp;</p>
     <p id="u335-232"><span id="u335-229">​</span><span><span class="actAsInlineDiv normal_text" id="u487"><!-- content --><span class="actAsDiv clip_frame excludeFromNormalFlow" id="u481"><!-- image --><img id="u481_img" src="images/ironwave.jpg" alt="" width="98" height="98"/></span></span></span><span></span></p>
     <p id="u335-234">MOLECULAR CLOCK OF FOLD STRUCTURES UNCOVER THE RISE OF PLANET OXYGENATION AND AEROBIC METABOLISM</p>
     <p id="u335-235">&nbsp;</p>
     <p id="u335-237">We identified an important recurrence in evolution of domain structures. This clock placed timelines of domain appearance in the geological record. Remarkably, we find that the rise of planetary oxygen began ~2.9 Gy ago and unfolded pathways of aerobic metabolism. The rise of planet oxygenation was gradual and was likely triggered by the appearance of Mn catalase, probably in response to large amounts of radiation-induced oxygen peroxides in ice depositions of the first major glaciations. The identification of the molecular culprit of planetary oxygen enhances our understanding of metallomes and ocean biogeochemistry.</p>
     <p id="u335-238">&nbsp;</p>
     <p id="u335-246">• Kim et al.&nbsp; (2012) Structure 20:67-76 (<a class="nonblock" href="http://www.ncbi.nlm.nih.gov/pubmed/22244756">abstract</a>)(<a class="nonblock" href="http://www.cell.com/structure/retrieve/pii/S0969212611004163">PDF</a>)</p>
     <p id="u335-254">• Saito (2012) Structure 20:1-2 (<a class="nonblock" href="http://www.ncbi.nlm.nih.gov/pubmed/22244750">abstract</a>)(<a class="nonblock" href="http://www.sciencedirect.com/science/article/pii/S0969212611004618">PDF</a>)</p>
     <p id="u335-262">• Jiang et al. (2012) PLoS Comp Biol 8:e1002426 (<a class="nonblock" href="http://www.ncbi.nlm.nih.gov/pubmed/22438800">abstract</a>)(<a class="nonblock" href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002426">PDF</a>)</p>
     <p id="u335-270">• Wang et al. (2011) Mol Biol Evol 28:567-582 (<a class="nonblock" href="http://www.ncbi.nlm.nih.gov/pubmed/20805191">abstract</a>)(<a class="nonblock" href="http://mbe.oxfordjournals.org/content/28/1/567.long">PDF</a>)</p>
     <p id="u335-278">• Dupont et al. (2010) Proc Natl Acad Sci USA 107:10567-10572 (<a class="nonblock" href="http://www.ncbi.nlm.nih.gov/pubmed/20498051">abstract</a>)(<a class="nonblock" href="http://www.pnas.org/content/107/23/10567.long">PDF</a>)</p>
     <p id="u335-283">• Researchers identify molecular culprit in rise of planet oxygenation (<a class="nonblock" href="http://news.illinois.edu/news/12/0110oxygen_GustavoCaetano-Anolles.html">press release</a>)</p>
     <p id="u335-287"><span id="u335-284">​</span><span id="u335-285"><span class="actAsInlineDiv normal_text" id="u1231"><!-- content --><a class="excludeFromNormalFlow" id="untanglingbiodiversity"></a></span></span><span id="u335-286"></span></p>
     <p id="u335-289">___________________________________________________________________________________</p>
     <p id="u335-290">&nbsp;</p>
     <p id="u335-294"><span id="u335-291">​</span><span><span class="actAsInlineDiv normal_text" id="u494"><!-- content --><span class="actAsDiv clip_frame excludeFromNormalFlow" id="u488"><!-- image --><img id="u488_img" src="images/fingerprinting2.jpg" alt="" width="98" height="98"/></span></span></span><span></span></p>
     <p id="u335-296">UNTANGLING BIODIVERSITY</p>
     <p id="u335-297">&nbsp;</p>
     <p id="u335-299">Molecular biotechnology provides tools for the evaluation and management of resources and change. Nucleic acid markers are important for such endeavor because they can measure constitution, diversity and evolution of genetic material, serving as suitable indicators of wealth, health and future of life. A number of techniques for genome analysis have been applied to the characterization of plant and microbes. These involve nucleic acid sequencing, amplification and hybridization. Some of these tools are currently being applied to study plant and microbial diversity and to determine the role of spontaneous mutation in evolution. We have also generated theoretical frameworks to explain hierarchical levels of structural organization that are widespread in biology and the diversity of organisms as these interact with the environment. Three persistence strategies, economy, flexibility and robustness, are used to explain the existence of a fully diversified tripartite cellular world and the existence of kingdoms of organisms in biology.</p>
     <p id="u335-300">&nbsp;</p>
     <p id="u335-308">• Yafremava et al. (2013) Front Gene 4: 16 (<a class="nonblock" href="http://www.ncbi.nlm.nih.gov/pubmed/23443991">abstract</a>)(<a class="nonblock" href="http://www.readcube.com/articles/10.3389/fgene.2013.00016">PDF</a>)</p>
     <p id="u335-316">• Mittenthal et al. (2012) Front Gene 3:147 (<a class="nonblock" href="http://www.ncbi.nlm.nih.gov/pubmed/22891076">abstract</a>)(<a class="nonblock" href="http://www.frontiersin.org/Systems_Biology/10.3389/fgene.2012.00147/abstract">PDF</a>)</p>
     <p id="u335-320"><span id="u335-317">​</span><span id="u335-318"><span class="actAsInlineDiv normal_text" id="u1233"><!-- content --><a class="excludeFromNormalFlow" id="crosstalkplantsmicroorganisms"></a></span></span><span id="u335-319"></span></p>
     <p id="u335-322">___________________________________________________________________________________</p>
     <p id="u335-323">&nbsp;</p>
     <p id="u335-327"><span id="u335-324">​</span><span><span class="actAsInlineDiv normal_text" id="u501"><!-- content --><span class="actAsDiv clip_frame excludeFromNormalFlow" id="u495"><!-- image --><img id="u495_img" src="images/f1large.jpg" alt="" width="99" height="98"/></span></span></span><span></span></p>
     <p id="u335-329">CROSSTALK DURING THE INTERACTION OF PLANTS AND MICROORGANISMS</p>
     <p id="u335-330">&nbsp;</p>
     <p id="u335-332">Legumes and bacteria (rhizobia) interact to establish a unique association that results in the formation of a highly specialized organ, the nitrogen-fixing nodule. This symbiosis is central to agronomical improvement because it provides a biological alternative to the use of nitrogen fertilizer in agriculture. To satisfy the expectations of a growing human population which will surely double in the next 40 years and currently demands 23 million tons of nitrogen per year, crop yield must increase without fertilizer-mediated pollution (leaching of toxic nitrates and ozone depletion). Therefore, biological nitrogen fixation offers the natural framework of a sustainable agricultural system and the possibilities of improvement of legumes through breeding and genetic engineering.</p>
     <p id="u335-333">&nbsp;</p>
     <p id="u335-335">Studies focus on the genes and molecular mechanisms that are responsible for switching on or off the cascade of developmental changes that occur when legume roots are infected by rhizobia, the symbiotic bacteria. Infection is initiated by rhizobial lipo-oligosaccharide signals. It results in cortical cell re-differentiation that leads to the formation of a new meristem which then develops into a nodule. Although the bacteria trigger this developmental sequence, it is the host which controls most facets of it. We have identified two very important genes, nnr and nts, that appear to be involved in these central aspects of host control. These genes are involved in signal transduction and control the efficiency with which nodule meristems progress into mature nodules.</p>
     <p id="u335-336">&nbsp;</p>
     <p id="u335-338">Other aspects of the signal interplay between microbes and plants are being investigated. Bacterial N-acyl homoserine lactone (AHL) signals coordinate the behavior of individual microbial cells. The successful infection of eukaryotic hosts by bacteria seems particularly dependent on such AHL-mediated “quorum sensing” regulation. The model legume Medicago truncatula detects nanomolar to micromolar levels of bacterial AHLs and responds to changes in the accumulation of hundreds proteins, by the activation of gusA reporter fusions in a gene and tissue-specific manner, and by altered secretion of compounds that mimic AHLs. Eukaryotes respond extensively and specifically to bacterial-quorum signals and this behavior could play important roles in the beneficial or pathogenic outcomes of the interaction between plants and microbes. We are currently exploring links between these responses and the developmental trigger of symbiotic and pathogenic responses.</p>
     <p id="u335-339">&nbsp;</p>
     <p id="u335-347">• Mathesius et al. (2003) Proc Natl Acad Sci USA 100:1444-1449 (<a class="nonblock" href="http://www.ncbi.nlm.nih.gov/pubmed/12511600">abstract</a>)(<a class="nonblock" href="http://www.pnas.org/content/100/3/1444.long">PDF</a>)</p>
     <p id="u335-349">• Caetano-Anollés (2001) Agro-Food-Industry Hi-Tech 12:32-35.</p>
     <p id="u335-351">• Caetano-Anollés (1997) Fields Crops Research 53:47-68.</p>
     <p id="u335-352">&nbsp;</p>
     <p id="u335-354">___________________________________________________________________________________</p>
     <p id="u335-358"><span id="u335-355">​</span><span><span class="actAsInlineDiv normal_text" id="u1141"><!-- content --><span class="actAsDiv clip_frame excludeFromNormalFlow" id="u1136"><!-- image --><img id="u1136_img" src="images/nekochas.gif" alt="" width="256" height="48"/></span></span></span></p>
    </div>
    <div class="verticalspacer"></div>
    <div class="clearfix colelem" id="pu321-4"><!-- group -->
     <div class="clearfix grpelem" id="u321-4"><!-- content -->
      <p><span id="u321">Gustavo Caetano-Anollés Lab - Evolutionary Bioinformatics at UIUC</span></p>
     </div>
     <div class="clearfix grpelem" id="u334-26"><!-- content -->
      <p><a class="nonblock" href="index.html">Home</a><span id="u334-3">&nbsp; /&nbsp; </span><a class="nonblock" href="research.html">Research</a><span id="u334-6">&nbsp; /&nbsp; </span><a class="nonblock" href="research.html">Resources</a><span id="u334-9">&nbsp; /&nbsp; </span><a class="nonblock" href="publications.html">Publications</a>&nbsp; <span id="u334-13">/&nbsp; </span><a class="nonblock" href="courses.html">Courses</a>&nbsp; <span id="u334-17">/&nbsp; </span><a class="nonblock" href="lab-members.html">Lab Members</a>&nbsp; <span id="u334-21">/&nbsp; </span><a class="nonblock" href="links.html">Links</a></p>
     </div>
    </div>
   </div>
  </div>
  <!-- JS includes -->
  <script type="text/javascript">
   if (document.location.protocol != 'https:') document.write('\x3Cscript src="http://musecdn2.businesscatalyst.com/scripts/4.0/jquery-1.8.3.min.js" type="text/javascript">\x3C/script>');
</script>
  <script type="text/javascript">
   window.jQuery || document.write('\x3Cscript src="scripts/jquery-1.8.3.min.js" type="text/javascript">\x3C/script>');
</script>
  <script src="scripts/museutils.js?275725342" type="text/javascript"></script>
  <script src="scripts/jquery.musemenu.js?4042164668" type="text/javascript"></script>
  <script src="scripts/jquery.watch.js?3999102769" type="text/javascript"></script>
  <!-- Other scripts -->
  <script type="text/javascript">
   $(document).ready(function() { try {
(function(){var a={},b=function(a){if(a.match(/^rgb/))return a=a.replace(/\s+/g,"").match(/([\d\,]+)/gi)[0].split(","),(parseInt(a[0])<<16)+(parseInt(a[1])<<8)+parseInt(a[2]);if(a.match(/^\#/))return parseInt(a.substr(1),16);return 0};(function(){$('link[type="text/css"]').each(function(){var b=($(this).attr("href")||"").match(/\/?css\/([\w\-]+\.css)\?(\d+)/);b&&b[1]&&b[2]&&(a[b[1]]=b[2])})})();(function(){$("body").append('<div class="version" style="display:none; width:1px; height:1px;"></div>');
for(var c=$(".version"),d=0;d<Muse.assets.required.length;){var f=Muse.assets.required[d],g=f.match(/([\w\-\.]+)\.(\w+)$/),k=g&&g[1]?g[1]:null,g=g&&g[2]?g[2]:null;switch(g.toLowerCase()){case "css":k=k.replace(/\W/gi,"_").replace(/^([^a-z])/gi,"_$1");c.addClass(k);var g=b(c.css("color")),h=b(c.css("background-color"));g!=0||h!=0?(Muse.assets.required.splice(d,1),"undefined"!=typeof a[f]&&(g!=a[f]>>>24||h!=(a[f]&16777215))&&Muse.assets.outOfDate.push(f)):d++;c.removeClass(k);break;case "js":k.match(/^jquery-[\d\.]+/gi)&&
typeof $!="undefined"?Muse.assets.required.splice(d,1):d++;break;default:throw Error("Unsupported file type: "+g);}}c.remove();if(Muse.assets.outOfDate.length||Muse.assets.required.length)c="Some files on the server may be missing or incorrect. Clear browser cache and try again. If the problem persists please contact website author.",(d=location&&location.search&&location.search.match&&location.search.match(/muse_debug/gi))&&Muse.assets.outOfDate.length&&(c+="\nOut of date: "+Muse.assets.outOfDate.join(",")),d&&Muse.assets.required.length&&(c+="\nMissing: "+Muse.assets.required.join(",")),alert(c)})()})();/* body */
Muse.Utils.transformMarkupToFixBrowserProblemsPreInit();/* body */
Muse.Utils.prepHyperlinks(true);/* body */
Muse.Utils.initWidget('.MenuBar', function(elem) { return $(elem).museMenu(); });/* unifiedNavBar */
Muse.Utils.fullPage('#page');/* 100% height page */
Muse.Utils.showWidgetsWhenReady();/* body */
Muse.Utils.transformMarkupToFixBrowserProblems();/* body */
} catch(e) { if (e && 'function' == typeof e.notify) e.notify(); else Muse.Assert.fail('Error calling selector function:' + e); }});
</script>
   </body>
</html>
